Makefile

```diff
@@ -1,12 +1,13 @@
 BIN="./bin"
 SRC=$(shell find . -name "*.go")
+GOPATH=$(shell go env GOPATH)
 
 ifeq (, $(shell which golangci-lint))
-$(warning "could not find golangci-lint in $(PATH), run: curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh")
+$(warning "could not find golangci-lint in PATH, run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOPATH)/bin v1.50.0")
```

Comment:
My only concern here is that the command now specifies a version of the linter to install (which would have to be kept up to date). Maybe it's better to simply point to the website which has good install docs: https://golangci-lint.run/usage/install/

Reply:
> My only concern here is that the command now specifies a version of the linter to install (which would have to be kept up to date). Maybe it's better to simply point to the website which has good install docs: https://golangci-lint.run/usage/install/

Agreed.

Reply:
An alternative is to use go install, as done in CI. However, that won't add it to the PATH.